Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["James Fairbanks", "Andrew Baas", "Evan Patterson", "Luke Morris", "G
 version = "0.5.0"
 
 [deps]
+ACSets = "227ef7b5-1206-438b-ac65-934d6da304b8"
```

Comment:
I don't see a compat for ACSets, but CompatHelper can catch this if it's not already in main.